src/styles/sections/_featured-speakers.scss

```diff
@@ -0,0 +1,80 @@
+.featured-speakers {
+  padding-block-end: 3rem;
+
+  .people {
+    list-style: none;
+
+    .speaker {
+      grid-template-columns: 1fr 3fr;
+      grid-template-rows: 1fr;
+
+      .left {
+        align-self: flex-start;
+        position: relative;
+
+        &::after {
+          content: "";
+          position: absolute;
+          height: 100%;
+          width: 100%;
+          top: -30%;
+          left: -30%;
+          background-image: url(@/assets/backgrounds/chessboard.svg);
+          background-repeat: no-repeat;
+          background-size: contain;
+          background-position: center;
+          z-index: -1;
+          filter: opacity(0.5);
+        }
```

Comment:
- [ ] Great job implementing this background grid in such a smart way. To match the original design, you should consider making it a bit smaller, maybe even a bit more transparent, and set a smaller offset.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142065347-d6fc3005-c252-4bde-a7cf-56f67aaf1f5b.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/142065394-529c47cc-8371-4ec6-9987-46c0f8f15e5e.png)

Reply:
Solved:

![image](https://user-images.githubusercontent.com/3728536/142073948-d9a4c6a0-8cfd-49f5-a833-2377bd95a1fb.png)